packages/front-end/components/Forms/Toggle.tsx

```diff
@@ -18,6 +19,7 @@ export default function Toggle({
   disabled?: boolean;
   type?: "featureValue" | "environment" | "toggle";
   className?: string;
+  style?: { [key: string]: string | number };
```

Comment:
There is a `CSSProperties` interface exported from `react` that should be used instead.

Reply:
I tried it, but got CI errors when using it.